debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java

```diff
@@ -608,7 +608,8 @@ private long asEpochMicros(String timestamp) {
                 new SchemaAndValueField("ts_large", MicroTimestamp.builder().optional().build(), expectedTsLarge),
                 new SchemaAndValueField("ts_large_us", MicroTimestamp.builder().optional().build(), expectedTsLargeUs),
                 new SchemaAndValueField("ts_large_ms", Timestamp.builder().optional().build(), expectedTsLargeMs),
-                new SchemaAndValueField("tz_large", ZonedTimestamp.builder().optional().build(), expectedTzLarge));
+                new SchemaAndValueField("tz_large", ZonedTimestamp.builder().optional().build(), expectedTzLarge),
+                new SchemaAndValueField("ts_pinf", MicroTimestamp.builder().optional().build(), Long.MAX_VALUE));
```

Comment:
Where's the values for ts_minf and tz_minf?

Reply:
Btw. how does this PR build not fail for decoderbufs, without the plug-in PR applied 🤔 ?

Reply:
It is easy, the CI build runs against `9.6-devel` tag that has the change applied